File: RSMK2/RSMK2/Action.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RSMK2
{
    [Serializable]
    public class Action
    {
        private string _name;
        private double _price;
        private double _time;
        private string _venik;

        public string GetName() => _name;
        public void SetName(string name) { _name = name; }
        public double GetPrice() => _price;
        public void SetPrice(double price) { _price = price; }
        public double GetTime() => _time;
        public void SetTime(double time) { _time = time; }
        public string GetBranch() => _venik;
        public void SetBranch(string tree) { _venik = tree; }

        public Action() { }
        public Action(string name, double price, double time, string venik)
        {
            _name = name;
            _price = price;
            _time = time;
            _venik = venik;
        }
    }
}

File: zwsserver/src/types/Action.java
package types;

public class Action {

    private String description;
    private double price;
    private double time;
    private String venik;

    public Action() {
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public void setPrice(double price) {
        this.price = price;
    }

    public void setTime(double time) {
        this.time = time;
    }

    public void setVenik(String venik) {
        this.venik = venik;
    }

    public Action(String description, double price, double time, String venik) {
        this.description = description;
        this.price = price;
        this.time = time;
        this.venik = venik;
    }

    public String getDescription() {
        return description;
    }

    public double getPrice() {
        return price;
    }

    public double getTime() {
        return time;
    }

    public String getVenik() {
        return venik;
    }

    @Override
    public String toString(){
        return description + "\t" + price + "\t" + time + "\t" + venik;
    }

}
